package.json

```diff
@@ -15,11 +15,13 @@
     "frontend_html": "script/vulcanize.js",
     "frontend_prod": "npm run js_prod && npm run frontend_html",
     "frontend_demo": "npm run js_demo && npm run frontend_html",
-    "ru_all": "npm run ru_core && npm run ru_demo",
+    "ru_all": "npm run ru_core && npm run ru_compatibility && npm run ru_demo",
     "ru_core": "rollup --config rollup/core.js",
+    "ru_compatibility": "rollup --config rollup/compatibility.js",
     "ru_demo": "rollup --config rollup/demo.js",
-    "watch_ru_all": "npm run watch_ru_core && npm run watch_ru_demo",
+    "watch_ru_all": "(npm run watch_ru_core & npm run watch_ru_compatibility & npm run watch_ru_demo) && wait",
```

Comment:
interesting, I never realized that. Then again, we hardly develop anything but the core.